www/dusv03/SP2/js/main.js

```diff
@@ -33,43 +27,32 @@ var searchValue;
 var url;
 var accessToken = params.access_token;
 var state = params.state;
-var userID;
 var stateKey = 'spotify_auth_state';
 var clientId = 'bd8138fa0b1640b297139a6b30482253';
-var redirectURI = "http://localhost:5500/4IZ268-2018-2019-ZS/www/dusv03/SP2/";
-var resArray = [];
+var redirectURI = "http://localhost:5500/";
```

Comment:
tohle prosim uprav na `https://fcp.vse.cz/4IZ268/2018-2019-ZS/www/dusv03/SP2` jinak to nebude fungovat na ostre verzi na internetu. taky to pridej do whitelistu na spotify API